src/utilities/input.tsx

```diff
@@ -43,7 +43,7 @@ type InputStyleGroup =
 export const InputStyles: Record<InputStyleGroup, string> = {
     base: 'tw-flex tw-items-center tw-justify-between tw-gap-2 tw-px-3 tw-transition tw-text-sm tw-font-sans tw-relative tw-bg-base tw-border tw-rounded tw-line-strong dark:tw-w-full dark:tw-flex dark:tw-items-center dark:tw-justify-between dark:tw-gap-2 dark:tw-px-3 dark:tw-transition dark:tw-text-sm dark:tw-font-sans dark:tw-relative dark:tw-bg-base dark:tw-border dark:tw-rounded dark:tw-line-strong',
     width: 'tw-w-full dark:tw-w-full',
-    height: 'tw-h-9 tw-min-h-[2.25rem] dark:tw-h-9 dark:tw-min-h-[2.25rem]',
```

Comment:
I think that it's not enough, I can still see the scrollbar. Maybe increase a tiny more.

Alos, I just noticed but I don't think you need to add the `dark:` prefix on everything but only on what's going to change, in this case only background and text color.

Reply:
Hmm I did increase it to `2.35rem` odd is was not committed.  Will address.

Reply:
I still see the scroll bar:

<img width="459" alt="Screenshot 2023-12-05 at 16 35 39" src="https://github.com/Frontify/fondue/assets/35405438/b4073808-f87e-4119-87ca-ae8da474b72c">

Reply:
I am not sure how?! I am not getting it.  😕 
![Screenshot 2023-12-05 at 16 38 23](https://github.com/Frontify/fondue/assets/47146443/7173b165-c46b-41be-8f3f-4c73328b0e6a)

Reply:
I use chrome. Am I the only one who sees it? I'm wondering.

Reply:
It's gone now. Weird. Ok, I'll approve the PR.

Reply:
I have checked on Firefox, Chrome and Edge and I can not see it. But you can't be the only one. 😁 I will do some more digging and see what edge case this could be.